src/containers/ProjectDashboardContainer.jsx

```diff
@@ -32,7 +32,7 @@ class ProjectDashboardContainer extends Component {
     return (
       <div>
         <h2>Project Dashboard</h2>
-        {React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
+        {project.primary_language && React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
```

Comment:
There's a default value for project set already, in the ducks file. This hack just prevents the app from crashing if you load one of the resource pages directly e.g.  if you've bookmarked the URL. 

I've got a better fix in mind but I didn't want to hold this PR up until I've got time to implement it.